### DISCUSSION TITLE
D13C17B0-7E57-7FD0-7A18-73907A81291C

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#107] [General]
# Kepler-22

Kepler-22 is a sun-like star about 638±3 light-years away in the constellation of Cygnus, that is orbited by a planet found to be unequivocally within the star's habitable zone.

https://celestia.mobi/addon?item=D13C17B0-7E57-7FD0-7A18-73907A81291C

**Comment by TheLostProbe** [comment]
kinda crazy how well PK managed to replicate the original texture of K-22b in that one illustration

**Comment by SilverXplors** [comment]
I wonder if this planet actually is an ocean planet as it is being depicted or it is a misunderstood mini-neptune. Btw this is my favorite exoplanet.

**Comment by thatonespacenerd** [comment]
Kepler 22b has been my favorite exoplanet for a long time so I recommend this add on
